### PR TITLE
more tests for DBAL/Types/CronExpressionType

### PR DESCRIPTION
**Pull request by Hanmac**
this is for more Coverage of the DBAL Type.
The getVarcharTypeDeclarationSQL was a bit tricky, so i just check if my length is in the returned string somewhere.

i removed the prophecy tiny part and used normal mock (like in the other tests)

### CODE REVIEW

File: src/Doctrine/DBAL/Types/CronExpressionType.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ public function getSQLDeclaration(array $column, AbstractPlatform $platform): st
 
     /**
      * @param mixed $value
+     * @throws ConversionException
```
Comment (loevgaard) [inline src/Doctrine/DBAL/Types/CronExpressionType.php:23]: This doesn't add any value

File: tests/Doctrine/DBAL/Types/CronExpressionTypeTest.php [hunk 2]
```diff
@@ -5,18 +5,75 @@
...
-use Prophecy\PhpUnit\ProphecyTrait;
 use Setono\CronExpressionBundle\Doctrine\DBAL\Types\CronExpressionType;
+use stdClass;
 
 final class CronExpressionTypeTest extends TestCase
 {
-    use ProphecyTrait;
+    /**
+     * @test
+     * @throws DbalException
```
Comment (loevgaard) [inline tests/Doctrine/DBAL/Types/CronExpressionTypeTest.php:20]: This doesn't add any value

**Comment by loevgaard** [comment]
Could you remove the `@throws` phpdocs that doesn't add any value? :)

**Comment by Hanmac** [comment]
> Could you remove the `@throws` phpdocs that doesn't add any value? :)

my IDE was showing warnings about them, so i tried to remove them

**Comment by loevgaard** [comment]
> my IDE was showing warnings about them, so i tried to remove them

I understand, but it doesn't really add any value. At least not when you're not calling the respective methods and your IDE would show that you have an uncaught exception

**Comment by Hanmac** [comment]
the Coding Standards Test is broken and i can't fix it i think

**Comment by loevgaard** [comment]
Yeah. I will fix that. Thank you, @Hanmac 🎉

**Comment by loevgaard** [comment]
Fixed here: https://github.com/Setono/CronExpressionBundle/commit/0d6cee814902f952e8263725d9c390ac18e23676

**Pull request merged**